### PR TITLE
F#src 2599 create squeleton

### PR DESCRIPTION
**Pull request by beatrizleon**

### CODE REVIEW

File: docs/user_guide/1_arm_and_hand.md [hunk 1]
```diff
@@ -0,0 +1,659 @@
...
+
+To quickly start up the robot after it has been installed, perform the following steps:
+1. Press the power button on the teach pendant.
+2. Wait a minute while the system is starting up, displaying text on the touch screen.
+3. Press ON button on the touch screen. Wait a few seconds until robot state changes to idle.
+4. Press START button on the touch screen. The robot now makes a sound and moves a little while releasing the brakes.
+
+#### Configuring the network
+
+In order to use the robot with our driver you need to change the network setup of the robot via the pedant, performing the following steps:
```
Comment (ethanfowler) [inline docs/user_guide/1_arm_and_hand.md]: Typo, pedant -> pendant.

Comment (ethanfowler) [reply docs/user_guide/1_arm_and_hand.md]: Or is this too pedantic of me? I'll see myself out...

File: docs/user_guide/1_arm_and_hand.md [hunk 2]
```diff
@@ -0,0 +1,659 @@
...
+   * IP address: 192.168.1.1
+   * Subnet mask: 255.255.255.0
+   
+   ```eval_rst
+   .. image:: ../img/configure_arm_4.jpg
+   ```
+4. Press "Apply" when you finish.
+
+#### Arm Calibration Procedure
+Follow this steps if you have a table and a stylus provided by Shadow Robot.
```
Comment (ethanfowler) [inline docs/user_guide/1_arm_and_hand.md]: Typo, this steps -> these steps.

File: docs/user_guide/1_arm_and_hand.md [hunk 3]
```diff
@@ -0,0 +1,659 @@
...
+```eval_rst
+.. image:: ../img/world_generator_gui.png
+  :width: 400
+  :align: center
+```
+
+ In order to start it, make sure no Gazebo sessions are up and run:
+
+```sh
+roslaunch sr_world_generator world_generator_gui.lauch
```
Comment (ethanfowler) [inline docs/user_guide/1_arm_and_hand.md]: Typo, .lauch > .launch

File: docs/user_guide/1_arm_and_hand.md (hunk 1 above)
Comment (beatrizleon) [reply docs/user_guide/1_arm_and_hand.md]: :P

File: docs/user_guide/1_arm_and_hand.md (hunk 2 above)
Comment (beatrizleon) [reply docs/user_guide/1_arm_and_hand.md]: Done

File: docs/user_guide/1_arm_and_hand.md (hunk 3 above)
Comment (beatrizleon) [reply docs/user_guide/1_arm_and_hand.md]: Done

File: docs/user_guide/1_arm_and_hand.md (hunk 1 above)
Comment (beatrizleon) [reply docs/user_guide/1_arm_and_hand.md]: Done

**Pull request merged**